src/app/home/home.component.html

```diff
@@ -3,36 +3,36 @@
         <img src="../../assets/CashmereLogo.svg" height="190px" width="auto" alt="Fabric.Cashmere Logo"/>
     </span>
     <span class="headerRight">
-        <img src="../../assets/FabricCashmere.svg" height="55px" width="auto" alt="Fabric.Cashmere Title"/>
+        <img src="../../assets/FabricCashmere.svg" height="100%" width="auto" alt="Fabric.Cashmere Title"/>
         <p class="headerCopy">Health Catalyst's living style guide for Angular</p>
         <button hc-button title="Primary Button" color="primary" routerLink="/demo">Get Started</button>
```

Comment:
Super minor, but while you're in there... could we change the title attribute on the buttons on this page to not be "Primary Button"? Was checking out the homepage and noticed the browser tooltips

Reply:
Ah that's a great catch!  I completely missed that.  Will do.